pkg/cloudprovider/providers/gce/gce_clusters.go

```diff
@@ -70,13 +77,18 @@ func (gce *GCECloud) listClustersInZone(zone string) ([]string, error) {
 	return result, nil
 }
 
-func (gce *GCECloud) getClustersInZone(zone string) ([]*container.Cluster, error) {
-	mc := newClustersMetricContext("list_zone", zone)
+func (gce *GCECloud) getClustersInLocation(zoneOrRegion string) ([]*container.Cluster, error) {
+	mc := newClustersMetricContext("list_zone", zoneOrRegion)
```

Comment:
"list_zone" now seems like an inaccurate name?

Reply:
It does. However I am worried about continuity for that metric. Filed https://github.com/kubernetes/kubernetes/issues/68913 to track getting this done as a follow on.